rosdep/python.yaml

```
@@ -2780,6 +2780,11 @@ python-pyinotify:
   fedora: [python-inotify]
   gentoo: [dev-python/pyinotify]
   ubuntu: [python-pyinotify]
+python-pyjwt-pip:
```

Since the package no longer uses pip, the suffix should be dropped.
```suggestion
python-jwt:
```